.github/workflows/push.yaml

```diff
@@ -16,4 +16,6 @@ jobs:
         with:
           name: ngi
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-      - run: nix flake check
+      - run: nix flake check --impure
```

Comment:
See comment on https://github.com/ngi-nix/ngipkgs/pull/90/files#diff-7b2a967de7060a238e49a57908f83e860b8cb53a0b0991c3d0d275857c1014d4R18

Reply:
The package with unfree deps has since been removed by us.

Reply:
Consider resolving.